content/en/docs/concepts/architecture/mixed-version-proxy.md

```diff
@@ -58,9 +58,9 @@ these fields in the API server configuration file.
 If these flags are unspecified, peers will use the value from either `--advertise-address` or
 `--bind-address` command line argument to the kube-apiserver. If those too, are unset, the host's default interface is used.
 
-## Mixed version proxying
+## Mixed Version Proxying
 
-When you enable mixed version proxying, the [aggregation layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) 
+When you enable {{< glossary_tooltip text="MVP" term_id="mixed-version-proxy" >}}, the [aggregation layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) 
```

Comment:
You don't need a tooltip in the page about the concept.

Reply:
Removed, I misunderstood from earlier that we wanted to use the tooltip in the concept page. Hence made all the references in here.